### ISSUE TITLE
angular js support

### ISSUE THREAD

**Issue by dejanlukic** [wet-boew/wet-boew#7469]
all looks great but in IE ng-change is not working.
are you planning to add support for it?

**Comment by LaurentGoderre** [comment]
I have used Angular with ng-changed before without any problem.. Can you please describe your problem more specifically.

**Comment by dejanlukic** [comment]
I have an angular controller with funcition searchaudits(....).
On the page I have following markup:

<td><input title="Modified To" class="table-header" type="date"
ng-model="CreatedTo"
ng-keyup="searchaudits(Event,User,CreatedFrom,CreatedTo,PackageName,SenderName,PackageStatus,Documents,Signers,Contractor,BrokerName,BrokerCity,PackageId)"
ng-change="searchaudits(Event,User,CreatedFrom,CreatedTo,PackageName,SenderName,PackageStatus,Documents,Signers,Contractor,BrokerName,BrokerCity,PackageId)"
/></td>

This works nicely in all browsers but IE since they have native support for
input type="date".
In IE (i'm using IE 10) calendar control pops up but after selecting date
ng-change doesn't fire up.

**Comment by LaurentGoderre** [comment]
This has been fixed and will be available in the coming version (it didn't work in Firefox as well).

Duplicated of #6683

**Comment by LaurentGoderre** [comment]
As a temporary workaround I used the following in my angular app

```
 wb.doc.on('blur', 'input[type=date]', function() {
    $(this).trigger("change");
});
```

**Comment by LaurentGoderre** [comment]
This is now included in v4.0.21
